content/docs/dvclive/api-reference/init.md

```diff
@@ -0,0 +1,61 @@
+# dvclive.init()
+
+Initializes a DVCLive logger.
+
+```py
+def init(
+    path: str = None,
+    resume: bool = False,
+    step: int = 0,
+    summary: bool = True,
+    html: bool = True,
+)
+```
+
+#### Usage:
+
+```py
+import dvclive
+
+dvclive.init()
+```
+
+## Description
+
+It's usage is optional and focused on configuring the behavior of subsequent
+calls to [`dvclive.log`](/doc/dvclive/api-reference/log) and
+[`dvclive.next_step`](/doc/dvclive/api-reference/next_step).
+
+⚠️ If `path` already exists when this functions is called, it's contents will be
+removed.
+
+## Parameters
+
+- `path` (`dvclive` by default) - The _metrics logs_ (generated by
```

Comment:
potential improvement: metrics logs, plots logs - etc - we should link or use "abbr" tooltips to explain them I think

Reply:
I tried to explain those concepts by linking them to `dvc plots` / `dvc metrics` and update some parts for consistency on the naming. Still not 100% convinced, though